README.md

```diff
@@ -21,4 +47,13 @@ This Python tool aims to detect and remove any of this potentially malicious con
 
 A command-line program written in modern Python (3.10+) that is capable of locating and removing macros and dynamic content from a variety of files.
 
-It should support all the common Office Open XML formats (e.g. '.pptx', .docx', '.xlsx', etc.) as well as the 'legacy' MS binary file formats, like '.doc' and '.ppt'.
+It should support all the common Office Open XML formats, 'legacy' MS binary files, and PDFs.
+
+
+## Usage
+
+This package has not yet been published. It can be used, however, by cloning the repo, opening the command prompt in it, and running the following command:
+
+```poetry run bleach <PATH_TO_FILE>```
```

Comment:
I will make some small edits, to show you to how make this easier on the user. I will add some commits for this later this evening.